packages/vest/src/core/suite/createSuite.js

```diff
@@ -67,6 +67,14 @@ const createSuite = withArgs(args => {
       }
     });
   });
+  suite.hasErrors = (fieldName) => suite.get().hasErrors(fieldName);
+  suite.hasWarnings = (fieldName) => suite.get().hasWarnings(fieldName);
+  suite.getErrors = (fieldName) => suite.get().getErrors(fieldName);
+  suite.getWarnings = (fieldName) => suite.get().getWarnings(fieldName);
+  suite.hasErrorsByGroup = (groupName, fieldName) => suite.get().hasErrorsByGroup(groupName, fieldName);
+  suite.hasWarningsByGroup = (groupName, fieldName) => suite.get().hasWarningsByGroup(groupName, fieldName);
+  suite.getErrorsByGroup = (groupName, fieldName) => suite.get().getErrorsByGroup(groupName, fieldName);
+  suite.getWarningsByGroup = (groupName, fieldName) => suite.get().getWarningsByGroup(groupName, fieldName);
```

Comment:
Nice! This looks pretty good. Let's also add a couple of tests for these added functions. Nothing big, we only need to check that they match the original implementation.

Also, do you want to handle the documentation as well or should I?